bedrock/firefox/redirects.py

```diff
@@ -520,4 +520,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 1299947
     redirect('^firefox/channel/?$', 'firefox.channel.desktop'),
+
+    # Bug 1277196
+    redirect('^firefox/firstrun/learnmore/$', 'firefox.features'),
+    redirect('^firefox/windows-10/welcome/$', 'https://support.mozilla.org/kb/how-change-your-default-browser-windows-10'),
```

Comment:
You need to make sure the last trailing slash is optional here: `r'^firefox/windows-10/welcome/?$'`